src/track/globaltrackcache.cpp

```diff
@@ -123,6 +123,11 @@ void GlobalTrackCacheLocker::purgeTrackId(const TrackId& trackId) {
     return m_pInstance->purgeTrackId(trackId);
 }
 
+void GlobalTrackCacheLocker::invalidateCache() const {
```

Comment:
Uhhh, I don't think that this is a good idea and cannot guarantee that it doesn't cause undefined behavior!! This code was never intended to be invoked while `Track` objects are allocated and still alive, but only as a last resort when shutting down.